frontend/src/pages/Common/CreationDialog.js

```diff
@@ -100,6 +100,7 @@ const CreationDialog = props => {
       data-test="creation-dialog"
     >
       <DialogTitle> {title}</DialogTitle>
+      {props.gdprWarning}
```

Comment:
I suggest to rename this variable since GDPR is europe specific. Maybe dataPrivacyWarning

Reply:
this variable is not in use, I missed to delete this.